.github/workflows/bundle-size.yml

```diff
@@ -10,14 +10,14 @@ jobs:
         runs-on: ubuntu-latest
 
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
             - name: Use desired version of NodeJS
-              uses: actions/setup-node@v2
+              uses: actions/setup-node@v3
               with:
                   node-version-file: '.nvmrc'
                   cache: npm
             - name: Checking for file changes
-              uses: preactjs/compressed-size-action@b03ec15e1a93cb5c062d66a1c263f9ddbd2f81ef
+              uses: preactjs/compressed-size-action@v2
```

Comment:
https://github.com/preactjs/compressed-size-action/commits/master

v2 will target v2.5.0, released before the previously targeted commit (`b03ec15e1a93cb5c062d66a1c263f9ddbd2f81ef`), so we are using an older version now.

By the way, this action generates another deprecating message related to NodeJS 12, if you look at the latest run's Summary.
The latest commit in the repo updated the action to run on NodeJS 16, so we could pin the action to this one.

Reply:
Great catch! Addressed in [2ea3e97](https://github.com/Automattic/woocommerce-payments/pull/5680/commits/2ea3e97ce8891eb96d7df0f0d6964e294fddae6c)